app/controllers/admin/todolist_controller.rb

```diff
@@ -1,6 +1,10 @@
 class Admin::TodolistController < Admin::AdminController
-
   def index
-    @petitions = Petition.for_state(Petition::SPONSORED_STATE).order(:created_at).paginate(:page => params[:page], :per_page => params[:per_page] || 20)
+    @petitions = Petition.in_moderation.order(:created_at)
+    if params[:petition] == 'collecting_sponsors'
+      @petitions = Petition.collecting_sponsors.order(:created_at)
+    end
+    @petitions = @petitions.paginate(page: params[:page], per_page: params[:per_page] || 20)
```

Comment:
Nitpick:  This might be clearer as:

```ruby
if params[:petition] == 'collecting_sponsors'
  @petitions = Petition.collecting_sponsors
else
  @petitions = Petition.in_moderation
end
@petitions = @petitions.order(:created_at).paginate(page: params[:page], per_page: params[:per_page] || 20)
```

There might even be a cleaner way to express it than that (not a huge fan of ``@petitions = @petitions.<blah>`` at the end.